filterparser.go

```diff
@@ -75,6 +75,7 @@ const (
 	wordNOT         = "NOT"
 	wordEXISTS      = "EXISTS"
 	wordNOTEXISTS   = "NOTEXISTS"
+	wordNOTEXISTSSP = "NOT EXISTS"
```

Comment:
let's add the space to `wordNOTEXISTS` and not duplicate the word

Reply:
✅ 

Donezo, good 👀!